dashboards-observability/public/components/visualizations/charts/bar/bar.tsx

```diff
@@ -68,11 +76,11 @@ export const Bar = ({ visualizations, layout, config }: any) => {
     PLOTLY_COLOR[index % PLOTLY_COLOR.length];
 
   const prepareData = (valueForXSeries) => {
-    return (valueForXSeries.map((dimension: any) => data[dimension.label]))?.reduce(
-      (prev, cur) => {
+    return valueForXSeries
```

Comment:
check file formatting (for all files.)with prettier once.

Reply:
Done